config/environments/development.rb

```diff
@@ -33,6 +33,8 @@
 
   config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
 
+  config.action_mailer.delivery_method = :letter_opener
```

Comment:
When I was doing request for membership emails, I set up mailtrap. See `config/initializers/mailer.rb` and the readme. How does this interact with mailtrap? Would this replace it?